liquibase-core/src/main/java/liquibase/sqlgenerator/core/SetViewRemarksGenerator.java

```diff
@@ -15,7 +15,7 @@ public class SetViewRemarksGenerator extends AbstractSqlGenerator<SetViewRemarks
 
     @Override
     public boolean supports(SetViewRemarksStatement statement, Database database) {
-        return (database instanceof OracleDatabase) || (database instanceof PostgresDatabase) || (database instanceof MSSQLDatabase);
+        return (database instanceof OracleDatabase) || (database instanceof PostgresDatabase) || (database instanceof MSSQLDatabase) || (database instanceof DB2Database);
```

Comment:
That's what we have now: instanceof instanceof instanceof instanceof .... But maybe in a future refactor we can use interfaces or methods to validate it, like database.isSupportsViewRemarks or something less instanceof .

Reply:
Good idea @filipelautert . Maybe you can log a tech debt story for this.